### DISCUSSION TITLE
PKC not syncing on Vero 4K+ and Vero V anymore

### DISCUSSION THREAD

**Discussion by Dok-Ondar** [croneter/PlexKodiConnect#2065] [Q&A] [answered]
Since the update that required to refresh the library fully (a few weeks ago), the automatic syncing doesn't work anymore. I have to restart the Vero to get it synced again. I even did a full fresh install of my Vero V.

It is definitely PKC that is not syncing, the Plex Server, if I log in through a web interface, is showing the correct library status and on the Plex add-on on my LG TV, it also works like expected. Both Vero's have the same issue with PKC.

Have always used PKC very satisfactory up until now, thanks for making this!
Any advice on this issue?

**Comment by croneter** [comment]
Please share a DEBUG log file and open a new issue, see https://github.com/croneter/PlexKodiConnect/wiki/How-to-Report-A-Bug

**Comment by Dok-Ondar** [comment]
Will make a bug report in one or two days. Used PM4K in the hope that would solve my issue. While it did solve this issue, it came with a load more than PKC doesn't. Then you know how good PKC actually is, top job!

Installed PKC again and just need a day to see the issue. Thanks for replying.

**Comment by croneter** [comment, marked as answer]
https://github.com/croneter/PlexKodiConnect/issues/2069
